Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -32,6 +33,7 @@ namespace Content.Server.Medical.BiomassReclaimer
 {
     public sealed class BiomassReclaimerSystem : EntitySystem
     {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
```

Comment:
Every entitysystem contains an implicit reference to `IEntityManager`, so this is not needed.